Makefile

```diff
@@ -44,7 +44,7 @@ generate: build # Generate CLI docs and samples
 	go generate ./...
 
 .PHONY: bindata
-OLM_VERSIONS = 0.16.1 0.17.0 0.18.2
+OLM_VERSIONS = 0.19.0 0.19.1 0.20.0
```

Comment:
It seems weird to have both `0.19.0` and `0.19.1`. I would have expected `0.18.3`, `0.19.1`, & `0.20.0`.

Reply:
updated in https://github.com/operator-framework/operator-sdk/pull/5684/commits/2fd0a1e3d372482d9497e84787fb27bca30e883a